src/renderer/components/configure-recording-set-page/configure-recording-set-page.test.tsx

```diff
@@ -1,19 +1,29 @@
-import { render, screen } from '@testing-library/react';
+import { cleanup, render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import '@testing-library/jest-dom/extend-expect';
 import React from 'react';
 import { noop } from 'rxjs';
 import { mocked } from 'ts-jest/utils';
 
 import recordingListDataService from '../../services/recording-list-data-service';
+import { openFilePicker } from '../../utils';
 
 import { ConfigureRecordingSetPage } from './configure-recording-set-page';
 
 jest.mock('../../services/recording-list-data-service');
 // Mock the remote module to prevent import failure.
 jest.mock('@electron/remote', () => ({ dialog: jest.fn() }));
+jest.mock('../../utils', () => ({
+  ...jest.requireActual('../../utils'),
+  openFilePicker: jest.fn().mockImplementation(),
```

Comment:
If there are no implementations, then you don't need to call `.mockImplementation()`.